packages/entity-server/src/routes/hierarchies/HierarchiesRoute.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+
+import { QueryConjunctions, Route } from '@tupaia/server-boilerplate';
+import { formatEntitiesForResponse } from './format';
+import { FlattenedHierarchy, HierarchyContext, HierarchyResponseObject } from './types';
+
+type ReqParams = Record<string, never>;
+type ResBody = HierarchyResponseObject[] | FlattenedHierarchy[];
+type ReqBody = Record<string, never>;
+
+interface ReqQuery {
+  fields?: string;
+  field?: string;
+}
+
+export interface HierarchyRequest extends Request<ReqParams, ResBody, ReqBody, ReqQuery> {
+  ctx: HierarchyContext;
+}
+
+export class HierarchyRoute extends Route<HierarchyRequest> {
+  public async buildResponse() {
+    const { field, fields } = this.req.ctx;
```

Comment:
Inconsistency with existing endpoint: `filter` is not supported.

To do this the "right way", extracting and refactor existing filtering functionality to generalise it may be required. I felt that it's not worth the effort at this stage